astrokat/test/testutils.py

```diff
@@ -0,0 +1,88 @@
+import logging
+import os
+from six.moves import StringIO
+
+from astrokat import observe_main, simulate, utility
+
+
+__all__ = ['yaml_path', 'LoggedTelescope']
```

Comment:
Is this still correct?  We're only using `execute_observe_main` and `LoggedTelescope` in the tests.

Reply:
Will drop dunder `__all__`, it is a nice to have but seems unnecessary at this time.